### ISSUE TITLE
Improve Command Prompt UX

### ISSUE THREAD

**Issue by mhils** [mitmproxy/mitmproxy#3289]
From justrs on Slack:

> I'm not a mitmproxy newbie but I installed a new version and I'm completely baffled. Can someone point me to how to edit a URL (or even see the full URL?) I load a flow and hit e -> d and "flow.set @focus url str" shows up, but I have no idea what to do from there and cannot find it in the docs anywhere.

This feedback makes sense and applies to all keyboard shortcuts which open a command prompt. As a user, I probably expect some form of editor popping up. To the user it's not really clear that a command prompt has just opened. Can we make this clearer somehow?

One possible idea would be to update the bar above from this:
![image](https://user-images.githubusercontent.com/1019198/44003346-932ed62e-9e51-11e8-86c1-7a77ee188bbc.png)
To this:
![image](https://user-images.githubusercontent.com/1019198/44003379-ef48bf2e-9e51-11e8-8d17-6c5e89efa6d2.png)

@cortesi, any particular opinion?

**Comment by Jessonsotoventura** [comment]
@mhils  I don't think this should be improved with more text. We already have ':' to indicate that we are in command mode. This change should be more visual. Below is a POC for an action bar that is:

1. The same color as the status bar by default. Causing the status bar to appear as two lines
2. Changes to the background color on focus

If you want to play with the change, have a look at the `clear_command` branch: https://github.com/Jessonsotoventura/mitmproxy/tree/clear_command 

![Example for when a message appears ](https://user-images.githubusercontent.com/6732578/61680049-414c2980-acd6-11e9-8835-29a36bc0f539.png)
![Example for when one_key like q is pressed ](https://user-images.githubusercontent.com/6732578/61680052-427d5680-acd6-11e9-8d38-80c368466503.png)
![Example of the changing action bar](https://user-images.githubusercontent.com/6732578/61680202-ba4b8100-acd6-11e9-9c91-fa6cb2a384bd.png)
![Example of the default statusbar state](https://user-images.githubusercontent.com/6732578/61680204-bc154480-acd6-11e9-8389-8847ff068be1.png)
